script.js

```diff
@@ -39,4 +36,175 @@ window.onresize = () => {
   }
 };
 
+// Model
+const overlayModal = document.getElementById('overlayModal');
+function openModal(modal) {
+  if (modal == null) return;
+  modal.classList.add('active');
+  window.scrollTo(0, 0);
+  overlayModal.classList.add('active');
+  navLinks.forEach((link) => {
+    link.style.color = '#c1c7d0';
+  });
+}
+
+function closeModal(modal) {
+  if (modal == null) return;
+  modal.classList.remove('active');
+  modal.innerHTML = '';
+
+  overlayModal.classList.remove('active');
+  navLinks.forEach((link) => {
+    if (w > 768) {
+      link.style.color = '#344563';
+    } else {
+      link.style.color = '#fff';
+    }
+  });
+}
```

Comment:
- [ ] You did a great job on the styles but for this project, you need to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=z855fWEmkYWYXoJx-0) design as much as possible. Please make sure the project title, tags, and image start on the same vertical line and reduce the gap between the project title and tags.

### Your design

![image](https://user-images.githubusercontent.com/83803414/217813675-90230244-9f73-477b-845c-b98407393bd1.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/217812091-3ca45b8f-8b1f-4aa6-959c-552f3adfe298.png)

Reply:
Fixed.